### DISCUSSION TITLE
Just to clarify, what does eager option do?

### DISCUSSION THREAD

**Discussion by Disorrder** [beholdr/maska#99] [Q&A] [answered]
I tried `eager` as true, false, null, undefined, it not affect mask behaviour.

P.S.: `reversed` option is also not so obvious, it doesn't reverse input or mask itself, but start checking from the end. Have to try to understand

**Comment by beholdr** [comment, marked as answer]
Reversed mean that masked symbols will be written eagerly, e.g. with mask `#-#` when you input `1` it will be

- `1` in normal mode
- `1-` in eager mode

I think I’ll clarify in docs.
